src/test/java/org/openmrs/contrib/qaframework/automation/PageRejectedException.java

```diff
@@ -27,5 +27,4 @@ public PageRejectedException(String message, Throwable cause,
 			boolean enableSuppression, boolean writableStackTrace) {
 		super(message, cause, enableSuppression, writableStackTrace);
 	}
-
```

Comment:
Do we also need empty line here

Reply:
> Do we also need empty line here

we do not need it and its why is removed!